src/test/java/seedu/agendum/logic/LogicManagerTest.java

```diff
@@ -203,38 +203,56 @@ public void execute_addDuplicate_notAllowed() throws Exception {
     public void execute_list_showsAllTasks() throws Exception {
         // prepare expectations
         TestDataHelper helper = new TestDataHelper();
-        ToDoList expectedAB = helper.generateToDoList(2);
-        List<? extends ReadOnlyTask> expectedList = expectedAB.getTaskList();
+        ToDoList expectedTDL = helper.generateToDoList(2);
+        List<? extends ReadOnlyTask> expectedList = expectedTDL.getTaskList();
 
         // prepare to do list state
         helper.addToModel(model, 2);
 
         assertCommandBehavior("list",
                 ListCommand.MESSAGE_SUCCESS,
-                expectedAB,
+                expectedTDL,
                 expectedList);
     }
 
 
     /**
-     * Confirms the 'invalid argument index number behaviour' for the given command
+     * Confirms the 'incorrect index format behaviour' for the given command
+     * targeting a single task in the shown list, using visible index.
+     * @param commandWord to test assuming it targets a single task in the last shown list based on visible index.
+     * @param wordsAfterIndex contains a string that will usually follow the command
+     * 
+     * This (overloaded) method is created for rename/schedule
+     */
+    private void assertIncorrectIndexFormatBehaviorForCommand(String commandWord, String expectedMessage, String wordsAfterIndex)
```

Comment:
is this okay? rename is similar to delete/mark/unmark etc now but it takes in words at the back (which needs to be present)